main.py

```diff
@@ -233,7 +260,10 @@ def cli(dry_run, verbose):
             )
             click.secho(f"\n{source}\n{stats}")
 
-        raise click.Abort
+            if use_sentry:
```

Comment:
nit: is it really necessary to have this `use_sentry` variable?  `if SENTRY_DSN` wouldn't be too bad IMO ;)

Reply:
Removed. 

I've tested it. If you don't have a `SENTRY_DSN` and thus never called `sentry_sdk.init()` all future calls to `sentry_sdk.capture_exception` or `sentry_sdk.capture_message` are just no-ops that do nothing and breaks nothing.